### ISSUE TITLE
startUserPlayback returns 400: Malformed Json

### ISSUE THREAD

**Issue by wmorawski** [calluswhatyouwant/spotify-web-sdk#77]
**Describe the bug**
Current implementation takes an array of uris as an argument of startUserPlayback and transform it to comma separated string. Spotify API accepts an array of strings, not comma separated string.

**To Reproduce**
1. Call `startUserPlayback()` method with some uris:
```js
startUserPlayback({ uris: ['spotify:track:3FaQgxX3YENJdih4LIQhFF', 'spotify:track:3SmUKvWEaTViTxmVYJzMIK'] });
```
2. Check network tab, you will see red call with response code 400
3. Body has beend transformed into:
```json
{"uris":"spotify:track:3FaQgxX3YENJdih4LIQhFF,spotify:track:3SmUKvWEaTViTxmVYJzMIK"}
```

**Expected behavior**
An array of uris should be send instead of string

**Screenshots**
1. Wrong parameters (copied cURL from network tab)
![error](https://i.imgur.com/7Nepx16.png "error")
2. Correct params
![success](https://i.imgur.com/yZmvsYY.png "success")

**Comment by JoseRenan** [comment]
Hmmm, I think Spotify changed recently the way they accept lists 🤔Thanks for reporting!!!! We will fix it ASAP

**Comment by JoseRenan** [comment]
Hi, @wmorawski, we fixed this issue on branch `v0.5.x`, we plan to release it on npm as soon as we fix a merge problem with the tests on the `master` branch, I'm working on it right now!
